src/main/resources/templates/swift/xcode/solution/${packageNameFolder}/SampleText.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+class SampleText {
+    var text: String
```

Comment:
Same here:
Shouldn't it be private, since it is stated as private in the Problem Statement?
```suggestion
    private var text: String
```